features/attached_commands.feature

```diff
@@ -494,18 +494,9 @@ Feature: Command behaviour when attached to an UA subscription
           metering_timer: 0
         """
         And I run `python3 /usr/lib/ubuntu-advantage/timer.py` with sudo
-        Then stderr matches regexp:
-        """
-        Invalid value for update_messaging interval found in config.
-        """
-        And stderr matches regexp:
-        """
-        Invalid value for update_status interval found in config.
-        """
-        And stderr does not match regexp:
-        """
-        Invalid value for gcp_auto_attach interval found in config.
-        """
+        Then I verify that running `grep "Invalid value for update_messaging interval found in config." /var/log/ubuntu-advantage.log` `with sudo` exits `0`
```

Comment:
We should change the path for the grep command, right ?

Reply:
Right, it just failed here - will do